game.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'item'
+class Game < Item
+  attr_accessor :multiplayer, :last_played_date
+
+  # rubocop:disable Metrics/ParameterLists
+  def initialize(name, published_date, multiplayer, last_played_date, id: Random.rand(1..1000), archived: false)
+    # rubocop:enable Metrics/ParameterLists
```

Comment:
- [ ] [OPTIONAL] Going forward, let's reduce the way in which we disable linters and learn to fix the errors manually. This will help you become a great programmer.